modules/survey_accounts/php/NDB_Menu_Filter_survey_accounts.class.inc

```diff
@@ -49,7 +49,8 @@ class NDB_Menu_Filter_Survey_Accounts extends NDB_Menu_Filter
         // the base query
         $query = " FROM participant_accounts p 
             JOIN session s ON (p.SessionID=s.ID)
-            JOIN candidate c ON (c.CandID=s.CandID)";
+            JOIN candidate c ON (c.CandID=s.CandID)
+            WHERE c.Active = 'Y'";
```

Comment:
shouldn't this also have s.Active='Y'?

Reply:
Leigh, could you comment on Dave's comment

Reply:
@PapillonMcGill Hi Melanie, I don't think Leigh is working wil GitHub. you should email her.